### ISSUE TITLE
Fix hard-coded UART path in a Virtualbox 6 compatible way

### ISSUE THREAD

**Issue by elguitar** [Seravo/wp-vagrant#65]
**NOTE: All of these are true for the ubuntu/bionic64 box too!**

## Problem
6b36249bced3b409f021de192e937bfb8cc858b6 was a mistake, but reverting it isn't the solution either,

After updating the box, `vagrant up` fails with a hardcoded path which is generated during building:
```
[:~/seravo/wordpress] master+* 3m42s ± vagrant up 
Bringing machine 'wordpress-box' up with 'virtualbox' provider...
==> wordpress-box: Importing base box 'seravo/wordpress'...
==> wordpress-box: Matching MAC address for NAT networking...
==> wordpress-box: Checking if box 'seravo/wordpress' version '20201127.0.0' is up to date...
==> wordpress-box: Setting the name of the VM: wordpress_wordpress-box_1606743429262_295
==> wordpress-box: Clearing any previously set network interfaces...
==> wordpress-box: Preparing network interfaces based on configuration...
    wordpress-box: Adapter 1: nat
    wordpress-box: Adapter 2: hostonly
==> wordpress-box: Forwarding ports...
    wordpress-box: 22 (guest) => 2222 (host) (adapter 1)
==> wordpress-box: Running 'pre-boot' VM customizations...
==> wordpress-box: Booting VM...
There was an error while executing `VBoxManage`, a CLI used by Vagrant
for controlling VirtualBox. The command and stderr is shown below.                   

Command: ["startvm", "7af72bad-205c-49bc-95c0-a3543973d433", "--type", "headless"]   

Stderr: VBoxManage: error: RawFile#0 failed to create the raw output file /home/otto/github-seravo/wp-vagrant/ubuntu-bionic-18.04-cloudimg-console.log (VERR_FILE_NOT_FOUND)                                                                                   
VBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole                                                                 
```

The path can be changed from Virtualbox's machine settings to something that exists and everything works out fine.

If the UART is disabled, the boot with Virtualbox 6 is slooooow (doesn't seem to boot at all). In https://github.com/hashicorp/vagrant/issues/10578 it is discussed, that `ubuntu/xenial64` prints error to serial device -log during boot time.

## Solutions

1. Change the base box to `bento/bionic64`. It doesn't output stuff to a serial device during boot time. There is a high risk of regressions. Changing the base box is not usually recommended.
1. Generate the log path in https://github.com/seravo/wordpress. This makes stuff incompatible with each other and the roll-out can be quite messy.

**Comment by elguitar** [comment]
We can toggle the boottime logging off during building?

**Comment by elguitar** [comment]
Working on this now

**Comment by elguitar** [comment]
Fixed via efb1405
